src/varlinkctl/varlinkctl.c

```diff
@@ -783,6 +783,8 @@ static int verb_call(int argc, char *argv[], void *userdata) {
                                          "Method call %s() returned expected error: %s", method, error);
 
                                 r = 0;
+                        } else if (streq(error, SD_VARLINK_ERROR_EXPECTED_MORE)) {
+                                r = log_error_errno(r, "Method call %s() failed: called without 'more' flag, but flag needs to be set.", method);
```

Comment:
Here, please do not pass `r` to log_error_errno(), but use `SYNTHETIC_ERRNO(EBADE)`.